data/net/minecraft/world/level/biome/BiomeGenerationSettings.mapping

```diff
@@ -36,3 +36,7 @@ CLASS net/minecraft/world/level/biome/BiomeGenerationSettings
 			ARG 2 feature
 		METHOD addFeatureStepsUpTo (I)V
 			ARG 1 step
+		METHOD lambda$addCarver$0 (Lnet/minecraft/world/level/levelgen/GenerationStep$Carving;)Ljava/util/List;
+			ARG 0 ignored
```

Comment:
```suggestion
			ARG 0 key
```

It's a `map.computeIfAbsent(key -> new ArrayList())`, which reads better than "ignored".

Reply:
I was kinda thinking that deliberately ignored params would be better as `ignored` to signify that they were deliberately ignored.

Reply:
I'd agree, if the parameter was otherwise unknown as to *why* it's ignored. E.g.

```java
// 'ignored' tells the caller they can pass whatever they want in here, which is useful information
void methodAModderMayWantToCall(int ignored) {}
```

But in this case, we're looking at the parameter in a lambda within a MC method. Which makes more sense to name it based on what is the most readable, in this case, `computeIfAbsent` takes a `key` and returns a new mapping.

```
map.computeIfAbsent(key -> new ArrayList<>()); // this reads better
map.computeIfAbsent(ignored -> new ArrayList<>()); // what does being `ignored` tell me, that is immediately obvious in this case.
```

Anecdotally, I've seen this exact pattern (with `_k`, `k`, `key` or similar), way more over a MC codebase + libraries than `ignored`.

Reply:
yep, ok sure.